paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutUI.kt

```diff
@@ -2,16 +2,50 @@ package com.stripe.android.paymentsheet.verticalmode
 
 import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
 import androidx.compose.material.Text
 import androidx.compose.material.TextButton
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.unit.dp
 import com.stripe.android.lpmfoundations.luxe.SupportedPaymentMethod
 import com.stripe.android.uicore.image.StripeImageLoader
 
+internal const val TEST_TAG_MANAGE_SCREEN = "TEST_TAG_MANAGE_SCREEN"
+
+@Composable
+internal fun PaymentMethodVerticalLayoutUI(interactor: PaymentMethodVerticalLayoutInteractor) {
+    val context = LocalContext.current
+    val imageLoader = remember {
+        StripeImageLoader(context.applicationContext)
+    }
+
+    val state by interactor.state.collectAsState()
```

Comment:
it seems like this is a common pattern, so this is a non-blocking comment/question: is combining the various states into one state the right thing to do here? E.g. if `isProcessing` changes, then this whole state changes and I would expect that we would re-render anything that depends on `state.supportedPaymentMethods`? Is that right? Should we change this?

Reply:
Compose should handle that! It'll rerun this code (but it's lightweight), and it'll see the params are unchanged, and it'll skip all the nested things!